apis/tests/llama_test.py

```diff
@@ -0,0 +1,11 @@
+from apis.api import ChatLLM
+chat = ChatLLM(
```

Comment:
Please refer to python test about how to write a Test. Basically, it should be 
```
Class TestXXX():
    pass
    def test_xxx():
        pass
```

Reply:
Please remove this to the top-level `ChatLLM/integration_tests/` which means you should create a new folder. 

Also add a test for chatGLM